### ISSUE TITLE
Web Interface Not Showing Data from Hooks

### ISSUE THREAD

**Issue by sangaline** [ac-pm/Inspeckage#64]
When I look at the Xposed logs, I can see evidence that the Inspeckage hooks are working

```
11-01 09:43:54.005 I/Xposed  (10273): Inspeckage_FileSystem:R/W [new File(String)]: /data/user/0/com.fastaccess.github
11-01 09:43:54.011 I/Xposed  (10273): Inspeckage_FileSystem:R/W [new File(String)]: /data/user/0/com.fastaccess.github/cache
11-01 09:43:54.014 I/Xposed  (10273): Inspeckage_FileSystem:R/W [new File(String)]: /data/user/0/com.fastaccess.github/cache/image_manager_disk_cache
11-01 09:43:54.041 I/Xposed  (10273): Inspeckage_FileSystem:R/W [new File(String)]: /data/user/0/com.fastaccess.github/shared_prefs
```

but none of these calls actually end up in the web interface. You can see here that I have auto-refresh on and that the XHR endpoints like `?type=file&value=fs&count=` are indeed being polled repeatedly.

![screenshot](https://user-images.githubusercontent.com/8236564/32278321-b5196b1e-beeb-11e7-916b-e2d114ec1369.png)

They just return nothing, even though Inspeckage is logging the hooked function calls.

This is using Xposed Framework version 88.2 with SDK version 24 on arm. The Inspeckage version is 2.3. I would be happy to provide any other debugging information that I can if you let me know what's relevant.

**Comment by dave08** [comment]
We're having the same problem...

**Comment by MuchiMuchiPink** [comment]
Yeah, Inspeckage hooks work only occasionally on Android 7.0 and Up. Some Apps works, others don't.
Firing up the same App on an older device with 6.0.1and the hooks work. I guess this is due to changes in ART.

**Comment by martin-schlossarek** [comment]
Same issue here. I have a old Genymotion virtual device with Android 6.0.0, Xposed 86, Inspeckage 2.0 and it works. With a new device with Android 6.0.0, Xposed 88.2 and Inspeckage 2.3 it does not work. All Tabs are empty.

**Comment by twocucao** [comment]
Same issue here. Android 6.0.0, Xposed 88.2 and Inspeckage 2.3 it does not work.

**Comment by Brubs-Rocks** [comment]
Hi guys, i had the same issue. It worked for me modifying the directory permission to 751 or even 677 from the app you are hooking. 
E.g:
chmod 751 /data/data/app package

**Comment by clviper** [comment]
This problem is due to the fact that additional security measures were implemented in Android 7 (                    //https://developer.android.com/about/versions/nougat/android-7.0-changes#permfilesys). Inspeckage is creating word readable files inside each application sandbox so that the inspeckage app is able to read those files, but since Android 7, even if a file is world readable, other apps will not be able to read it. I'l try to do a pull request a possible fix for this problem.

**Comment by 5murfette** [comment]
why not optionally use su to access the files? I'm on Nox6+Inspeckage2.3, set chmod -R 777 /data/data/myapp and its still stuck on loading.... perhaps add more debug strings to help other figure out the issue?

**Comment by ninad-mathpati** [comment]
I tried on Android 9 and 10, even by providing permission its not working. Is there any other solution on this?
